src/gpuarray_blas_cuda_cublas.c

```diff
@@ -53,7 +53,8 @@ static const char *estr(cublasStatus_t err) {
 
 static inline int error_cublas(error *e, const char *msg, cublasStatus_t err) {
   return error_fmt(e, (err == CUBLAS_STATUS_ARCH_MISMATCH) ? GA_DEVSUP_ERROR : GA_BLAS_ERROR,
-                   "%s: %s", msg, estr(err));
+                   "%s: %s %s", msg, estr(err),
```

Comment:
This add an extra space when the old message is used. Can you remove the space on this line and add it in the string bellow?

Reply:
done